pkg/chart/chart.go

```diff
@@ -30,12 +39,12 @@ func ReadChartConfig(basePath string, chartName string) ([]byte, error) {
 // Read reads a README YAML file and returns the Content to render.
 func ReadChartMetadata(basePath string, chartName string) (Metadata, error) {
 	var m Metadata
-	chartPath := basePath + "/helm/" + chartName + "/Chart.yaml"
+	chartPath := basePath + HELM_CHARTS_FOLDER + chartName + "/Chart.yaml"
 
 	log.Printf("INFO - chart %s - reading Chart yaml", chartPath)
 	metadata, err := os.ReadFile(chartPath)
 	if err != nil {
-		return m, microerror.Maskf(CouldNotReadChartFileError, err.Error())
```

Comment:
Given https://github.com/giantswarm/helm-chart-docs-generator/pull/11/files#r1510904209, this error message doesn't make sense to me - the above lines read the metadata, no?